core/trino-main/src/main/java/io/trino/security/AccessControl.java

```diff
@@ -525,6 +525,13 @@ void checkCanRevokeRoles(SecurityContext context,
      */
     void checkCanExecuteFunction(SecurityContext context, String functionName);
```

Comment:
Should we deprecate or remove this one?

Reply:
No. This will be possible only after all functions gain some namespace, which i guess will happen rather sooner than later, but did not happen yet.